packages/documentation/copy/en/developer/Downloading and Building.md

```diff
@@ -1,34 +1,47 @@
 ---
-title: Downloading
+title: Developer Setup
 layout: docs
-permalink: /docs/handbook/download
-oneline: "Quick start with Lingua Franca."
+permalink: /docs/handbook/developer-setup
+oneline: "Setting up Lingua Franca for developers."
 preamble: >
 ---
 
-## Releases and Nightly Snapshots
+# Cloning the Repository
 
-To get started with Lingua Franca immediately, choose one or more of the following tools:
+```
+git clone git@github.com:lf-lang/lingua-franca.git
+cd lingua-franca
+git submodule update --init --recursive
+```
 
-- [Visual Studio Code Extension](/docs/handbook/code-extension), a popular IDE from Microsoft.
-- [Epoch IDE](/docs/handbook/epoch-ide), an Eclipse-based IDE specifically for Lingua Franca.
-- [Command-line Tools](/docs/handbook/command-line-tools)
-- [Vim and Neovim plugins](https://github.com/lf-lang/lingua-franca.vim)
+# Building from the Command Line
 
-The VS Code plugin gives you the latest **release** of the Lingua Franca tools, including the diagram synthesis.
-If instead you want to use the **most recent development version** of Lingua Franca, then you will be given the option to download the Epoch IDE or the command-line tools from the nightly build. Just follow the above links.
-The Vim plugin provides syntax-directed editing, but it does not include the diagram synthesis that is available with Epoch and Code.
-The diagram synthesis is quite useful, so we recommend using Epoch or Code to develop LF programs.
+## Command Line Tools
 
-## The Repository
+The command line tools are used using Gradle with `./gradlew buildAll`.
 
-If you plan to contribute to Lingua Franca, or if you want to keep up to date as the project evolves, you will need to work from the git repository on GitHub. There are several ways to do this:
+You can the Lingua Franca compiler and other CLI tools. For instance: `./bin/lfc --version`.
```

Comment:
This sentence is incomplete. Not sure what it intends to say.  Should we list the command line tools?